seldon-core-operator/rockcraft.yaml

```diff
@@ -12,7 +12,8 @@ description: |
 
 version: v1.15.0_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
```

Comment:
Although not touched by this PR, but can you update the comment here?

The value is not KF version in this case, so let's have the comment depict the actual scheme used for the name

Reply:
Done.